src/plugins/autoplay/autoplay.js

```diff
@@ -88,7 +110,18 @@
         }
 
         if ( timeout > 0 ) {
-            timeoutHandle = setTimeout( function() { api.next(); }, timeout * 1000 );
+            timeoutHandle = setTimeout( function() {
+                if (status === "delayed") {
+                    if (userSkipAutoplay > 0) {
+                        userSkipAutoplay = userSkipAutoplay - 1;
+                        setAutoplayTimeout( timeout );
```

Comment:
I don't understand this... If I have

    data-autoplay="5" data-autoplay-userdelay="2"

...then the total time on the slide is set to 15 seconds? This is a very weird outcome.

A simpler alternative, both for users and as implementation would be to just have userdelay as a boolean:

    data-autoplay="5" data-autoplay-userdelay="1"    (or "true")

and then in `userActivity()` just reset the clock:

    var userActivity = function(event) {
        reloadTimeout(event);
    }

You could make this enabled by default. It seems like a good feature.